packages/ui/src/propPanels/text.tsx

```diff
@@ -0,0 +1,139 @@
+import type { PropPanelSchema, PropPanelWidgetProps } from '../types'
+import React from 'react';
+import { Select, Checkbox, Form, InputNumber, Col, Row } from 'antd';
+import type { CheckboxChangeEvent } from 'antd/es/checkbox';
+import {
+    getFallbackFontName,
+    DEFAULT_FONT_NAME,
+    DEFAULT_FONT_SIZE,
+    DEFAULT_ALIGNMENT,
+    DEFAULT_VERTICAL_ALIGNMENT,
+    DEFAULT_CHARACTER_SPACING,
+    DEFAULT_LINE_HEIGHT,
+} from '@pdfme/common';
+
+const textSchema: Record<string, PropPanelSchema> = {
+    fontName: { title: 'Font Name', type: 'string', widget: 'FontSelect', span: 8 },
+    alignment: {
+        title: 'Text Align',
+        type: 'string',
+        widget: 'select',
+        props: {
+            options: [
+                { label: 'Left', value: 'left' },
```

Comment:
Do these labels work with translations? or does that need to be re-added?

Reply:
we need to be re-added.

Reply:
Ah, I see. Great work by the way 👍

Reply:
I plan to make all labels translatable in the future, as mentioned in the following issue.

https://github.com/pdfme/pdfme/issues/107